### ISSUE TITLE
Issue with Cost Accessibility Matrix

### ISSUE THREAD

**Issue by mc10-c** [Esri/public-transit-tools#198]
I have been attempting to use the Cost Accessibility Matrix with a Transit Network Dataset, a service area of census blocks, and 12 destination points. No matter what settings I adjust I repeatedly receive the error message "All OD Cost Matrix solves failed, so no output was produced" and therefore no transit accessibility calculations are produced. At the same time, according to the process messages, inputs are successfully validated, origins are copied to the output, inputs are successfully preprocessed polygons converted for OD Cost Matrix, and all the chunks of the OD Cost Matrix are solved in parallel and finished. Then I receive the attached messages. Also, I'd like to point out that the processing speed is around 1.5 minutes which I assume is too quick. Any tips on how to proceed?

![image](https://github.com/user-attachments/assets/59ddc772-c2d7-49a2-8d72-0b6ebbe5034b)

Thanks,
M

**Comment by mmorang** [comment]
Hello M.  Happy to help.

The messages you're seeing are "correct" in that the tool should report that it has completed an OD Cost Matrix calculation even if the solve failed because no solution was found (meaning that no origin-destination pairs could be reached within the travel time limit).  Then, you see that warning messages because all the solves failed, so there are no results for the entire combined problem.

So, what we need to figure out is why you're none of the OD pairs are getting a solution.  Some possibilities:
1. Your network dataset isn't well connected or isn't really working correctly.  Have you tested it manually by solving some simple point-to-point routes and Service Areas in the map just to make sure it's working right?  The most common problems here would be the wrong connectivity policy for streets or some problem setting up your cost or restriction attributes.
2. The date/time you used for the analysis isn't within the valid date range for the GTFS data you used, or maybe you need to use a specific date instead of a generic weekday.
3. There is legitimately no solution. Maybe your origins and destinations are just truly unreachable.
4. Maybe you've used the wrong time units somewhere, so you think you're putting in 45 minutes, but it's actually using a cutoff of 45 seconds, or something along those lines.

Does any of that give you any clues?  If none of that leads you to a solution, please send more info about your analysis settings, network dataset settings, etc, and we can dig deeper.
